# Holds on to the list of masters generated by the MMArbitrator file.
string[] robotKeys